proto/merge.go

```diff
@@ -257,8 +257,8 @@ func (p *descriptorProcessor) collectFDs() {
 		p.fds = append(p.fds, fd)
 	}
 
-	slices.SortFunc(p.fds, func(x, y *descriptorpb.FileDescriptorProto) int {
-		return strings.Compare(*x.Name, *y.Name)
+	sort.Slice(p.fds, func(x, y int) bool {
```

Comment:
Apparently, slices.SortFunc is more performant, but this is done in a go routine, so we probably don't really care??

Alternative is to bump to go 1.21, but that won't solve the compatibility issues with v0.47 (min go 1.19)